2014-12-18 10:54 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 10:54 INFO    : Allowing connection from ('127.0.0.1', 49702) (tcpserver.py:66)
2014-12-18 10:54 INFO    : Added client 822960, name  (object_sharer.py:80)
2014-12-18 10:54 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 10:54 INFO    : Allowing connection from ('127.0.0.1', 49704) (tcpserver.py:66)
2014-12-18 10:54 INFO    : Added client 262599, name  (object_sharer.py:80)
2014-12-18 11:03 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 11:03 INFO    : Allowing connection from ('127.0.0.1', 49717) (tcpserver.py:66)
2014-12-18 11:03 INFO    : Added client 548834, name  (object_sharer.py:80)
2014-12-18 11:07 INFO    : Emitting close-gui signal (qtflow.py:278)
2014-12-18 11:07 WARNING : Send exception ([Errno 10054] An existing connection was forcibly closed by the remote host), assuming client disconnected (object_sharer.py:352)
2014-12-18 11:07 WARNING : Client disconnected, removing (object_sharer.py:111)
2014-12-18 11:09 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 11:16 INFO    : Emitting close-gui signal (qtflow.py:278)
2014-12-18 11:16 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 11:17 INFO    : Emitting close-gui signal (qtflow.py:278)
2014-12-18 11:17 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 11:17 INFO    : Agilent_VNA_E5071C : Initializing instrument (Agilent_VNA_E5071C.py:47)
2014-12-18 11:17 ERROR   : Error creating instrument vna (instruments.py:295)
2014-12-18 11:17 INFO    : Object with name instrument_vna exists, replacing (object_sharer.py:138)
2014-12-18 11:22 INFO    : Emitting close-gui signal (qtflow.py:278)
2014-12-18 11:22 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 11:22 INFO    : Agilent_VNA_E5071C : Initializing instrument (Agilent_VNA_E5071C.py:47)
2014-12-18 11:22 ERROR   : Error creating instrument vna (instruments.py:295)
2014-12-18 11:22 INFO    : Object with name instrument_vna exists, replacing (object_sharer.py:138)
2014-12-18 11:33 INFO    : Emitting close-gui signal (qtflow.py:278)
2014-12-18 15:30 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 15:30 INFO    : Agilent_VNA_E5071C : Initializing instrument (Agilent_VNA_E5071C.py:47)
2014-12-18 15:30 ERROR   : Error creating instrument vna (instruments.py:295)
2014-12-18 15:30 INFO    : Object with name instrument_vna exists, replacing (object_sharer.py:138)
2014-12-18 15:30 INFO    : Emitting close-gui signal (qtflow.py:278)
2014-12-18 15:30 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 15:30 INFO    : Agilent_VNA_E5071C : Initializing instrument (Agilent_VNA_E5071C.py:47)
2014-12-18 15:30 ERROR   : Error creating instrument vna (instruments.py:295)
2014-12-18 15:30 INFO    : Object with name instrument_vna exists, replacing (object_sharer.py:138)
2014-12-18 15:31 INFO    : Emitting close-gui signal (qtflow.py:278)
2014-12-18 15:32 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 15:32 INFO    : Emitting close-gui signal (qtflow.py:278)
2014-12-18 15:32 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 15:32 INFO    : Emitting close-gui signal (qtflow.py:278)
2014-12-18 15:33 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 15:33 INFO    : Agilent_VNA_E5071C : Initializing instrument (Agilent_VNA_E5071C.py:47)
2014-12-18 15:33 ERROR   : Error creating instrument vna (instruments.py:295)
2014-12-18 15:33 INFO    : Object with name instrument_vna exists, replacing (object_sharer.py:138)
2014-12-18 15:34 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 15:34 INFO    : Agilent_VNA_E5071C : Initializing instrument (Agilent_VNA_E5071C.py:47)
2014-12-18 15:34 ERROR   : Error creating instrument vna (instruments.py:295)
2014-12-18 15:34 INFO    : Object with name instrument_vna exists, replacing (object_sharer.py:138)
2014-12-18 15:34 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 15:34 INFO    : Agilent_VNA_E5071C : Initializing instrument (Agilent_VNA_E5071C.py:47)
2014-12-18 15:34 ERROR   : Error creating instrument vna (instruments.py:295)
2014-12-18 15:34 INFO    : Object with name instrument_vna exists, replacing (object_sharer.py:138)
2014-12-18 15:34 INFO    : Emitting close-gui signal (qtflow.py:278)
2014-12-18 15:37 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 15:37 INFO    : Agilent_VNA_E5071C : Initializing instrument (Agilent_VNA_E5071C.py:47)
2014-12-18 15:37 ERROR   : Error creating instrument vna (instruments.py:295)
2014-12-18 15:37 INFO    : Object with name instrument_vna exists, replacing (object_sharer.py:138)
2014-12-18 15:38 INFO    : Emitting close-gui signal (qtflow.py:278)
2014-12-18 15:38 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 15:38 INFO    : Emitting close-gui signal (qtflow.py:278)
2014-12-18 15:40 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 15:41 INFO    : Agilent_VNA_E5071C : Initializing instrument (Agilent_VNA_E5071C.py:47)
2014-12-18 15:41 ERROR   : Error creating instrument vna (instruments.py:295)
2014-12-18 15:41 INFO    : Object with name instrument_vna exists, replacing (object_sharer.py:138)
2014-12-18 15:41 WARNING : Instrument "vna" already exists, removing (instruments.py:279)
2014-12-18 15:41 INFO    : Agilent_VNA_E5071C : Initializing instrument (Agilent_VNA_E5071C.py:47)
2014-12-18 15:41 INFO    : Object with name instrument_vna exists, replacing (object_sharer.py:138)
2014-12-18 15:41 ERROR   : Error creating instrument vna (instruments.py:295)
2014-12-18 15:41 INFO    : Object with name instrument_vna exists, replacing (object_sharer.py:138)
2014-12-18 15:44 INFO    : Emitting close-gui signal (qtflow.py:278)
2014-12-18 15:44 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 15:44 INFO    : Emitting close-gui signal (qtflow.py:278)
2014-12-18 15:44 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 15:44 INFO    : Agilent_VNA_E5071C : Initializing instrument (Agilent_VNA_E5071C.py:47)
2014-12-18 15:44 ERROR   : Error creating instrument vna (instruments.py:295)
2014-12-18 15:44 INFO    : Object with name instrument_vna exists, replacing (object_sharer.py:138)
2014-12-18 17:31 INFO    : Emitting close-gui signal (qtflow.py:278)
2014-12-18 17:34 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 17:34 INFO    : Agilent_VNA_E5071C : Initializing instrument (Agilent_VNA_E5071C.py:47)
2014-12-18 17:34 ERROR   : Error creating instrument vna (instruments.py:295)
2014-12-18 17:34 INFO    : Object with name instrument_vna exists, replacing (object_sharer.py:138)
2014-12-18 17:35 WARNING : Instrument "vna" already exists, removing (instruments.py:279)
2014-12-18 17:35 INFO    : Agilent_VNA_E5071C : Initializing instrument (Agilent_VNA_E5071C.py:47)
2014-12-18 17:35 INFO    : Object with name instrument_vna exists, replacing (object_sharer.py:138)
2014-12-18 17:35 ERROR   : Error creating instrument vna (instruments.py:295)
2014-12-18 17:35 INFO    : Object with name instrument_vna exists, replacing (object_sharer.py:138)
2014-12-18 17:35 INFO    : Emitting close-gui signal (qtflow.py:278)
2014-12-18 17:42 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 18:10 INFO    : Emitting close-gui signal (qtflow.py:278)
2014-12-18 18:15 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 18:21 INFO    : Emitting close-gui signal (qtflow.py:278)
2014-12-18 18:22 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 18:31 INFO    : Emitting close-gui signal (qtflow.py:278)
2014-12-18 18:31 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 18:31 INFO    : IVVIDIG_main : Initializing instrument IVVIDIG (IVVIDIG_main.py:46)
2014-12-18 18:31 ERROR   : Error creating instrument IVVI (instruments.py:295)
2014-12-18 18:31 INFO    : Object with name instrument_IVVI exists, replacing (object_sharer.py:138)
2014-12-18 18:31 INFO    : IVVIDIG_main : Deleting IVVI instrument (IVVIDIG_main.py:70)
2014-12-18 18:32 INFO    : Emitting close-gui signal (qtflow.py:278)
2014-12-18 18:32 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2014-12-18 18:35 INFO    : Agilent_VNA_E5071C : Initializing instrument (Agilent_VNA_E5071C.py:47)
2014-12-18 18:39 INFO    : IVVIDIG_main : Initializing instrument IVVIDIG (IVVIDIG_main.py:46)
2014-12-18 18:39 ERROR   : Error creating instrument IVVI (instruments.py:295)
2014-12-18 18:39 INFO    : Object with name instrument_IVVI exists, replacing (object_sharer.py:138)
2014-12-18 18:39 INFO    : IVVIDIG_main : Deleting IVVI instrument (IVVIDIG_main.py:70)
2014-12-18 18:56 INFO    : Emitting close-gui signal (qtflow.py:278)
2014-12-18 19:02 INFO    : psyco acceleration not enabled (02_qtlab_start.py:51)
2017-05-10 11:13 WARNING : Unable to load config file (config.py:98)
2017-05-10 11:13 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:13 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:13 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:13 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:13 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:13 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:13 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:13 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:13 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:13 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:13 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:13 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:13 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:13 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:13 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:13 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:38 WARNING : Unable to load config file (config.py:98)
2017-05-10 11:38 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:38 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:38 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:38 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:38 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:38 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:38 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:38 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:38 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:38 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:38 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:38 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:38 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:38 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:38 WARNING : Unable to save config file (config.py:139)
2017-05-10 11:38 WARNING : Unable to save config file (config.py:139)
2017-05-10 16:56 WARNING : Unable to load config file (config.py:98)
2017-05-10 16:56 WARNING : Unable to save config file (config.py:139)
2017-05-10 16:56 WARNING : Unable to save config file (config.py:139)
2017-05-10 16:56 WARNING : Unable to save config file (config.py:139)
2017-05-10 16:56 WARNING : Unable to save config file (config.py:139)
2017-05-10 16:56 WARNING : Unable to save config file (config.py:139)
2017-05-10 16:56 WARNING : Unable to save config file (config.py:139)
2017-05-10 16:56 WARNING : Unable to save config file (config.py:139)
2017-05-10 16:56 WARNING : Unable to save config file (config.py:139)
2017-05-10 16:56 WARNING : Unable to save config file (config.py:139)
2017-05-10 16:56 WARNING : Unable to save config file (config.py:139)
2017-05-10 16:56 WARNING : Unable to save config file (config.py:139)
2017-05-10 16:56 WARNING : Unable to save config file (config.py:139)
2017-05-10 16:56 WARNING : Unable to save config file (config.py:139)
2017-05-10 16:56 WARNING : Unable to save config file (config.py:139)
2017-05-10 16:56 WARNING : Unable to save config file (config.py:139)
2017-05-10 16:56 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:22 WARNING : Unable to load config file (config.py:98)
2017-05-11 09:22 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:22 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:22 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:22 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:22 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:22 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:22 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:22 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:22 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:22 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:22 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:22 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:22 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:22 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:22 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:22 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:54 WARNING : Unable to load config file (config.py:98)
2017-05-11 09:54 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:54 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:54 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:54 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:54 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:54 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:54 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:54 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:54 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:54 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:54 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:54 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:54 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:54 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:54 WARNING : Unable to save config file (config.py:139)
2017-05-11 09:54 WARNING : Unable to save config file (config.py:139)
2017-05-11 10:08 WARNING : Unable to load config file (config.py:98)
2017-05-11 10:08 WARNING : Unable to save config file (config.py:139)
2017-05-11 10:08 WARNING : Unable to save config file (config.py:139)
2017-05-11 10:08 WARNING : Unable to save config file (config.py:139)
2017-05-11 10:08 WARNING : Unable to save config file (config.py:139)
2017-05-11 10:08 WARNING : Unable to save config file (config.py:139)
2017-05-11 10:08 WARNING : Unable to save config file (config.py:139)
2017-05-11 10:08 WARNING : Unable to save config file (config.py:139)
2017-05-11 10:08 WARNING : Unable to save config file (config.py:139)
2017-05-11 10:08 WARNING : Unable to save config file (config.py:139)
2017-05-11 10:08 WARNING : Unable to save config file (config.py:139)
2017-05-11 10:08 WARNING : Unable to save config file (config.py:139)
2017-05-11 10:08 WARNING : Unable to save config file (config.py:139)
2017-05-11 10:08 WARNING : Unable to save config file (config.py:139)
2017-05-11 10:08 WARNING : Unable to save config file (config.py:139)
2017-05-11 10:08 WARNING : Unable to save config file (config.py:139)
2017-05-11 10:08 WARNING : Unable to save config file (config.py:139)
